app/controllers/api_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ApiController < ApplicationController
+  skip_before_filter  :verify_authenticity_token
```

Comment:
Hmmm, that makes sense, but it seems like it would be confusing to have some parts of the API one authentication and then another part another auth strategy.  I imagine there's a way to combine.  Maybe just check for the cookie property and if it matches up, check the token, and assume its browser based, otherwise, use an auth token.  Sort of what I did, just bringing back in the CSRF check if needed.